### DISCUSSION TITLE
1270/how-to-create-a-single-node-graylog-instance-and-analyze-fortigate-logs/

### DISCUSSION THREAD

**Discussion by giscus** [seanthegeek/seanthegeek.github.io#2] [giscus]
# 1270/how-to-create-a-single-node-graylog-instance-and-analyze-fortigate-logs/

A complete guide to creating a single-node Graylog instance, sending FortiGate firewall logs to it, and analyzing the data

https://seanthegeek.net/1270/how-to-create-a-single-node-graylog-instance-and-analyze-fortigate-logs/

**Comment by ffeingol** [comment]
Thanks for this really comprehensive guide.  How, exactly are individual fields created?  I can see the pipelines that clean them up, but not how they are created in the first place.  Thanks in advance.

**Reply by seanthegeek** [reply]
The syslog messages that are sent from the FortiGate have a `field=value` structure. Graylog knows the fields names from that, but not what those field types are, so it treats every field value as a string. The pipeline tells Graylog to treat specifically named fields as integers, etc.

**Comment by loyiso89** [comment]
Thanks Sean for this comprehensive guide, if you are using no TLS oon your server  and not tcp rather udp.
How can u tacle that one.
Please advise

**Reply by seanthegeek** [reply]
I do not recommend this because the data is sent in plain text. That said.

In Graylog: Navigate to System> Inputs. Launch a new Syslog UDP input. Set the Time zone to UTC. Clink Launch Input.

In the FortiGate CLI:

```text
config log syslogd setting
    set status enable
    set server "graylog.example.com"
    set mode udp
    set port 514
end
```

**Comment by kusannaghi** [comment]
I follow all instruction, but when i restart nginx I get a 502 error . In my graylog server I try curl -I http://127.0.0.1:9000 and i get "curl: (7) Failed to connect to 127.0.0.1 port 9000 after 0 ms: Couldn't connect to server". My graylog not show error, but i can't load the service. It's frustrating. Thanks
